code/modules/core_implant/cruciform/rituals/machinery.dm

```diff
@@ -30,7 +30,18 @@
 		fail("Cloner is off.", user, C)
 		return FALSE
 
+<<<<<<< HEAD
 	pod.start()
+=======
+	if(pod.start())
+		var/damage_modifier = 1
+		var/obj/item/weapon/implant/core_implant/cruciform/C_user = user.get_core_implant(/obj/item/weapon/implant/core_implant/cruciform)
+		if(C_user.get_module(CRUCIFORM_INQUISITOR) || (C_user.get_module(CRUCIFORM_PRIEST) && C_user.get_module(CRUCIFORM_REDLIGHT)))
+			damage_modifier = 0
+		else if(C_user.get_module(CRUCIFORM_PRIEST))
+			damage_modifier = 0.5
+		pod.clone_damage = clone_damage * damage_modifier
+>>>>>>> f79716f7f... Changes to cruciform energy regeneration and rituals. (#5975)
```

Comment:
I think you forgot to fix a conflict

Reply:
This file is disabled. We don't use these litanies.